catalog/spatial/registry/pom.xml

```diff
@@ -92,6 +92,11 @@
                 <version>${project.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.codice.ddf.registry</groupId>
```

Comment:
https://github.com/vinamartin/ddf/blob/337ab1095abec38c370f1fd6e63406b7632e7346/catalog/spatial/registry/pom.xml
See conversation with @shaundmorris above. After that, let me know if it still isn't correct.